lib/logstash.js

```diff
@@ -0,0 +1,79 @@
+var _ = require('underscore'),
+    sortObject = require('sort-object-keys');
+
+var keys = [
+    '@timestamp',
+    'level',
+    '@message',
+    'message',
+    'clientip',
+    'sessionID',
+    'hostname',
+    'port',
+    'method',
+    'path',
+    'httpversion',
+    'response',
+    'responseTime',
+    'bytes',
+    'clientip',
+    'sessionID',
+    'uniqueID',
+    'host',
+    'pm',
+    'label',
+    '@fields',
+    'error',
+    'stack',
+    'type'
+];
+
+var serialize = function (data) {
+    var seen = [];
+    data = sortObject(data, keys);
+    return JSON.stringify(data, function (k, v) {
+        if (typeof v === 'function') {
+            return '[func]';
+        }
+        if (typeof v === 'object' && v) {
+            if (seen.indexOf(v) >=0) { return '[circular]'; }
+            seen.push(v);
+            return _.isArray(v) ? v : sortObject(v, keys);
+        }
+        return v;
+    });
+};
+
+module.exports = function (options) {
```

Comment:
We could throw an error if no options are passed but it will error when it tries t access options.meta. This log formatter is only called by winston's internals so it  should always have options